### ISSUE TITLE
Question: Handling multiple logins simultaneously possible?

### ISSUE THREAD

**Issue by Ralms** [AzureAD/microsoft-authentication-library-for-js#478]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[x] Other... Please describe: Question
</code></pre>

## Library version

<pre><code>
Library version:  @azure/msal-angular: 0.1.2

</code></pre>

Question:
I've been exploring this library and I've managed to login into 2 accounts at the same time, however when I do getUser() I only get returned the last one.
Doing loginPopup() again will list me the logged in accounts if I pick a different one will automatically switch not requiring login again. 
So my question is that if I keep track of some user information such as all userIdentifiers that logged in, is there a way to define which user I want to do aquireTokenSilent() for example or other library method ? 

Thanks

**Comment by nehaagrawal** [comment]
@Ralms did you try getAllUsers()?

**Comment by Ralms** [comment]
@nehaagrawal 
Hi there,
Wasn't aware of that method.

Just tried it and worked 100%. 
Tested doing a login with 2 users, do getAllUsers().
On the resultant array of users I got a token by doing acquireTokenSilent(["user.read"], undefined, user)
And finally tested that token calling Graph https://graph.microsoft.com/v1.0/me 

Very happy that this can be achieved this easy where I can leave the entire token management part to the library. 

Thank you very much.
